.eslintrc

```diff
@@ -1,22 +1,29 @@
 rules:
-  indent:
-    - 2
-    - 4
-  quotes:
-    - 1
-    - single
   linebreak-style:
     - 2
     - unix
-  semi:
-    - 2
-    - always
-  comma-dangle:
-    - 1
-    - always-multiline
-  no-extra-semi:
+  no-console:
     - 1
 env:
   browser: true
   jquery: true
-extends: 'eslint:recommended'
+globals:
+  $: false
+  alert: true
+  Analytics: false
+  BugLinks: true
```

Comment:
Ideally, I'd prefer these globals to be specified on a per-file basis. Reason being, these aren't automatically global across the whole site, they're global on pages that include the correct bundles that define them. I dunno how many files we'd have to annotate, though; if it's a ton, it might not be worth changing.

Some of these, like the `$` for jQuery, are included in the `crashstats_base` bundle ([defined here](https://github.com/mozilla-services/socorro/blob/master/webapp-django/crashstats/settings/bundles.py#L360), and included in [the base template here](https://github.com/mozilla-services/socorro/blob/master/webapp-django/crashstats/base/jinja2/crashstats_base.html#L194)) and can be safely included here since they're on _every_ page.